airflow/api/common/experimental/get_code.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.exceptions import AirflowException, DagNotFound
+from airflow.models import DagBag
+from airflow.www import utils as wwwutils
+
+
+def get_code(dag_id):
+    """Return python code of a given dag_id."""
+    dagbag = DagBag()
```

Comment:
This is a potentially expensive operation (as it will load all dags from disk!)

Could you instead do something like in #4407 where we load the dag model from the DB instead.

Reply:
Thanks @ashb . Incorporated it

Reply:
@ashb could you please review it and provide your feedback/comments